CommonWallet/Classes/Common/Fee/FeeCalculationStrategy.swift

```diff
@@ -0,0 +1,26 @@
+/**
+* Copyright Soramitsu Co., Ltd. All Rights Reserved.
+* SPDX-License-Identifier: GPL-3.0
+*/
+
+import Foundation
+
+public protocol FeeCalculationStrategyProtocol {
+    func calculate(for amount: Decimal) throws -> Decimal
```

Comment:
Why throws?

Reply:
To have an opportunity to throw an error for invalid input.

Reply:
But it's decimal in input. And the realizations don't throw as well

Reply:
One can imagine cases that a value should be not less than a minimal one.